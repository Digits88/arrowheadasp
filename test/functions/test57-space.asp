<!-- Test Space function.
-->

Space(0) = "<%=Space(0)%>"<br>
Space(1) = "<%=Space(1)%>"<br>
Space(2) = "<%=Space(2)%>"<br>
Space(20) = "<%=Space(20)%>"<br>
