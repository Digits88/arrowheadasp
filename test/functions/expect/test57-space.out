<!-- Test Space function.
-->

Space(0) = ""<br>
Space(1) = " "<br>
Space(2) = "  "<br>
Space(20) = "                    "<br>
